src/api/app/controllers/webui/project_controller.rb

```diff
@@ -115,9 +135,13 @@ def new_incident
     end
   end
 
-  def new_package; end
+  def new_package
+    authorize @project, :update?
```

Comment:
are you sure we should add it here? With that, if the user hasn't authorization, she will be redirected to the main page, changing how it works today: https://21838-2046192-gh.circle-artifacts.com/0/capybara/Projects_creating_packages_in_projects_not_owned_by_user__eg__global_namespace_as_non_admin_user.png

@hennevogel

Reply:
Sure, you can't do anything on `new_package` anything without logging in or having access right?

Reply:
@hennevogel yes, however is the default behavior on obs to show all the views and do not hide anything from the user. Using the usability joker ("from the usability perspective..."), I think it's better to keep the user on the same screen instead of send him to the main index. What do you think @hennevogel ?